packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -207,36 +208,76 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
   );
 
   if (invitableExistingUsers.length) {
-    const organization = ctx.user.profile.organization;
-    const orgSlug = organization ? organization.slug || organization.requestedSlug : null;
     await handleExistingUsersInvites({
       invitableExistingUsers,
       team,
       orgConnectInfoByUsernameOrEmail,
-      input,
-      inviter: ctx.user,
+      teamId: team.id,
+      language,
+      isOrg: isTeamAnOrg,
+      inviter,
       orgSlug,
     });
   }
 
   if (IS_TEAM_BILLING_ENABLED) {
-    await updateQuantitySubscriptionFromStripe(team.parentId ?? input.teamId);
+    await updateQuantitySubscriptionFromStripe(team.parentId ?? team.id);
   }
 
   return {
-    ...input,
+    // TODO: Better rename it to invitations only maybe?
+    usernameOrEmail:
```

Comment:
Not sending the entire input, sending just what is required. Also, necessary now because this function isn't aware of what an input is now.